app/pages/preview/ui.test.js

```diff
@@ -18,7 +18,7 @@ const mocks = (existingData) => {
 
 const pageSetup = async (t, existingData = false) => {
   mocks(existingData);
-  await t.navigateTo(`${clientLocalhost}/preview`);
```

Comment:
Do you need to change this back?

Reply:
That's a good point. How is this test even passing 🤔

Reply:
oh yeah I deliberately changed it to a hardcoded path. I've changed `clientLocalhost` to include `/supplier` since that was the smallest change. I'm going to this properly once I add `/authority` so will look at doing it then. Is it ok to stay as is for now?